module-system/sov-modules-api/Cargo.toml

```diff
@@ -28,20 +28,20 @@ hex = { workspace = true, optional = true }
 clap = { workspace = true, optional = true }
 schemars = { workspace = true, optional = true, features = [] }
 
-# TODO: try update 2.x
-ed25519-dalek = { version = "1.0.1", default-features = false, features = ["u64_backend"] }
-rand = { version = "0.7", optional = true }
+ed25519-dalek = { version = "2.0.0", default-features = false }
```

Comment:
I noticed the `fast` feature while glancing at the 2.0 manifest. Could it be of interest to us?

https://github.com/dalek-cryptography/ed25519-dalek/blob/02001d8c3422fb0314b541fdb09d04760f7ab4ba/Cargo.toml#L65

Reply:
Interesting! @dubbelosix have you tried this one? I am not sure if it is going to be faster in ZK mode, but maybe in native?

CC @preston-evans98

Reply:
I'll turn this into an issue, so we can unblock this PR.